modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx

```diff
@@ -96,7 +96,11 @@ export function ModalAddDefaultSortColumn({
 	const onSubmit = (event: FormEvent) => {
 		event.preventDefault();
 
-		const objectFieldName = selectedObjectSortColumn?.objectFieldName;
+		let objectFieldName = selectedObjectSortColumn?.objectFieldName;
+
+		typeof objectFieldName === 'undefined' &&
+			filtredObjectSortColumn.length > 0 &&
+			(objectFieldName = filtredObjectSortColumn[0].objectFieldName);
```

Comment:
Hey @thienquach87 and @SamZiemer, I think we can make this code more readable, keeping your logic, you can add this in an `if` condition, like:

```ts
if (!objectFieldName && !!filtredObjectSortColumn.length) {
	objectFieldName = filtredObjectSortColumn[0].objectFieldName;
}
```

Reply:
Hi, @SamZiemer could you help adjust the code as @danvb112 suggested on this PR directly? I verified this logic is similar to my idea.

Reply:
@danvb112 I created new PR, please help check it
https://github.com/liferay-objects/liferay-portal/pull/807